openapi/sms/v1/instructions.md

```diff
@@ -0,0 +1,244 @@
+| `https://anrjzyscq.accounts400.ondemand.com/oauth2/certs` | SAP IAS JSON Web Key Set (JWKS) certificate information endpoint |
+
+---
+
+## Workflows
+
+Below are the supported workflows of a partner service
+
+| Workflow          | Required            |
+| ----------------- | ------------------- |
+| Client onboarding | <center>✅</center> |
+| Outbound messages | <center>✅</center> |
+| Inbound Messages  | <center>❌</center> |
+
+### Client Onboarding
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant Client as Client User
+  participant Emarsys as Emarsys SMS
+  participant Partner as Partner Service
+  Note right of Client: Onboarding
+  Client->>Emarsys: Client Configuration
+  Emarsys->>Partner: Client configuration
+  Note right of Partner: Partner stores the configuration used
+
+```
+
+A partner service requires the following endpoints to support client onboarding:
+
+---
+
+- #### [Create/Update Client Config](../partner-service.yaml/paths/~1clients~1{clientId}~1configuration/get)
+
+##### Request Schema:
+
+```yaml json_schema
+$ref: '../partner-service.yaml#/components/schemas/CreateOrUpdateClientConfigurationRequest'
+```
+
+##### Response Schema:
+
+```yaml json_schema
+$ref: '../partner-service.yaml#/components/schemas/CreateOrUpdateClientConfigurationResponse'
+```
+
+---
+
+- [Get Client Config](../partner-service.yaml/paths/~1clients~1{clientId}~1configuration/get)
+
+##### Response Schema:
+
+```yaml json_schema
+$ref: '../partner-service.yaml#/components/schemas/GetClientConfigurationResponse'
+```
+
+---
+
+- [Delete Client Config](../partner-service.yaml/paths/~1clients~1{clientId}~1configuration/delete)
+
+---
+
+### Outbound Messaging and Reporting
+
+#### Overview
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant Client as Client User
+  participant Emarsys as Emarsys SMS
+  participant Partner as Partner Service
+  participant Vendor as SMS Vendor API
+  loop Each SMS campaign trigger
+    Client-->Emarsys: Triggers Campaign
+    Emarsys->>Partner: Outbound Messages
+    Partner->>Vendor: Outbound SMS Messages
+    loop Each Delivery Report
+      Vendor->>Partner: SMS Delivery Receipt
+      Partner->>Emarsys: Message Delivery Report
+      Emarsys->>Client: Message Reporting
+      Note left of Client: User views reports via UI
+    end
+  end
+```
+
+#### Outbound Messages
+
+```mermaid
+sequenceDiagram
+  autonumber
+  participant Client as Client User
+  participant Emarsys as Emarsys SMS
+  participant Partner as Partner Service
+  participant Vendor as SMS Vendor API
+  loop Each SMS campaign trigger
+    Client-->Emarsys: Triggers Campaign
+    Emarsys->>Partner: Outbound Messages
+    Partner->>Vendor: Outbound SMS Messages
+  end
+```
+
+- [Create Outbound Message Batch](../partner-service.yaml/paths/~1clients~1{clientId}~1messages/post)
+  - Request Payload
+    ```yaml json_schema
+    $ref: '../partner-service.yaml#/components/schemas/OutboundMessageBatch'
```

Comment:
```suggestion
    $ref: './partner-service.yaml#/components/schemas/OutboundMessageBatch'
```

Reply:
we navigate to the parent directory here because in stoplight markdown files are places in a `docs` directory (within the root) whereas the OpenAPI yaml files are directly in the root